Extensions/tweaks.js

```diff
@@ -801,14 +807,21 @@ XKit.extensions.tweaks = new Object({
 				box-shadow: none !important;
 			}`, 'xkit_tweaks_hide_post_highlight');
 		}
-		
+
 		if (XKit.extensions.tweaks.preferences.grayscale_new_post_button.value) {
 			let postIconButtonSel = XKit.css_map.keyToClasses('postIconButton').map(cssClass => `.${cssClass} span`).join(',');
 			XKit.extensions.tweaks.add_css(`${postIconButtonSel} {
 				filter: grayscale(100%);
 			}`, 'xkit_tweaks_grayscale_new_post_button');
 		}
-		
+
+		if (XKit.extensions.tweaks.preferences.subtle_follow_button.value) {
+			let followButtonSelector = XKit.css_map.keyToClasses('followButton').map(cssClass => `.${cssClass}`).join(',');
```

Comment:
You can just use `keyToCss` here, since that does all the mapping work for you

Reply:
ah i had tried that and it didn't work... lemme try again i guess

Reply:
ah i was using it incorrectly before. updated!